BaseTools/Conf/tools_def.template

```diff
@@ -152,15 +159,17 @@ DEFINE DTC_BIN                 = ENV(DTC_PREFIX)dtc
 #                        Note:
 #                             Building of XIP firmware images for ARM/ARM64 is not currently supported (only applications).
 #                             /FILEALIGN:4096 and other changes are needed for ARM firmware builds.
-#   VS2019      -win32-  Requires:
+#   VS2019      -win32,win64-  Requires:

```

Comment:
This was a correction to the VS2019. VS2019 already had support for win64. If you would prefer that its another commit, I can still do that.

Reply:
> This was a correction to the VS2019. VS2019 already had support for win64. If you would prefer that its another commit, I can still do that.

It would be nice to have different commits for VS2019 vs VS2022 changes if you don't mind.

Reply:
Dropping the VS2019 chang for now.